catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/docs/slack-component.adoc

```diff
@@ -164,6 +168,35 @@ A CamelContext with Blueprint could be as:
 </blueprint>
 ---------------------------------------------------------------------------------------------------------------------------
 
+== Producer
+
+*Since Camel 3.9.0* +
```

Comment:
The since is not needed here. The documentation will be only for latest, so just remove the since, because the documentation is versioned